app/views/account/appointments/_form.html.erb

```diff
@@ -96,7 +96,7 @@
         </div>
 
         <div class="form-group mb-2">
-          <label class="form-label" for="appointment_comment">Optional: Tell us about the project you are working on. This may help us recommend a different or additional tool for you.</label>
+          <label class="form-label" for="appointment_comment">Optional: Tell us about the project you're working on. We may be able to recommend a different or additional tool. If you're dropping off, please let us know if you had any issues with the items you're returning.</label>
```

Comment:
Let's remove the "Optional:" prefix to the label. Even though this field is technically optional, we want members to typically fill it out so that library staff have more information and can be more helpful.